Dockerfile

```diff
@@ -63,9 +63,22 @@ ARG TARGETPLATFORM
 # lld has issues building static binaries for ppc so prefer ld for it
 RUN set -e; xx-apk add musl-dev gcc libseccomp-dev libseccomp-static; \
   [ "$(xx-info arch)" != "ppc64le" ] || XX_CC_PREFER_LINKER=ld xx-clang --setup-target-triple
-RUN --mount=from=runc-src,src=/usr/src/runc,target=. \
+RUN --mount=from=runc-src,src=/usr/src/runc,target=. --mount=type=tmpfs,target=libcontainer/dmz/binary \
   --mount=target=/root/.cache,type=cache <<EOT
   set -ex
+  if [ -f libcontainer/dmz/_dmz.c ]; then
+  case "$(xx-info arch)" in
+    amd64|arm64|s390x|riscv64|386 )
```

Comment:
That's not quite the same list as https://github.com/opencontainers/runc/blob/v1.2.0-rc.2/libcontainer/dmz/Makefile#L8 but arm & ppc64le give errors to me. Not sure if Alpine, clang or I'm passing some wrong flags.